.travis.yml

```diff
@@ -20,19 +34,49 @@ install:
 - rvm use 2.2.8 --install --fuzzy
 - gem update --system
 - gem install sass
-- gem install jekyll -v 3.2.1
+- gem install ruby_dep -v 1.3.1
+- gem install jekyll -v 3.4.3
 - gem install jekyll-redirect-from
 
 before_script:
   - sudo chmod +x /usr/local/bin/sbt
 
 script:
-- sbt ++$TRAVIS_SCALA_VERSION test:fastOptJS
-- sbt ++$TRAVIS_SCALA_VERSION orgScriptCI
+- if [ "$SCALAENV" = "jvm" ]; then
+    sbt ++$TRAVIS_SCALA_VERSION validateJVM;
+  elif [ "$SCALAENV" = "js" ]; then
+    sbt ++$TRAVIS_SCALA_VERSION compile;
+    sbt ++$TRAVIS_SCALA_VERSION validateJS;
+  elif [ "$G4SBUILD" = "docs" ]; then
+    sbt ++$TRAVIS_SCALA_VERSION docs/tut;
+  else
+    echo "You might not be invited to the party";
+  fi
 
 after_success:
 - bash <(curl -s https://codecov.io/bash)
-- sbt ++$TRAVIS_SCALA_VERSION orgAfterCISuccess
+
+jobs:
+  include:
+    - stage: deploy
+      script:
+        - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
+            if grep -q "SNAPSHOT" version.sbt; then
+              sbt ++$TRAVIS_SCALA_VERSION publish;
+            else
+              if [ "$SCALAENV" = "all" ]; then
```

Comment:
isn't this always the case due to the `env` on line 79?

Reply:
Yes, it should be